tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/backportIssue.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/issueManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/repoContentManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/retrier"
+	gogithub "github.com/google/go-github/v48/github"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	// Flag Names
+	toplevelIssueIdFlag = "toplevelIssueId"
+	backportVersionFlag = "backportVersions"
+
+	// Other constants
+	issueAutocreatedTemplate   = "Issue was generated by eksGoTools createCveIssue, by:"
+	eksGoOwner                 = "aws"
+	eksGoRepo                  = "eks-distro-build-tooling"
+	eksGoSupportedVersionsPath = "projects/golang/go/MAINTAINED_EOL_VERSIONS"
+)
+
+var (
+	toplevelIssueId  int
+	backportVersions []string
+
+	backportIssueCmd = &cobra.Command{
+		Use:   "issue",
+		Short: "Opens backport issues for top level github issue",
+		Long:  `Opens issues to backport top level issue to EKS-Distro supported versions of Golang`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			label := []string{"golang", "security"}
+			assignee := "rcrozean"
+			issueState := "open"
+
+			retrier := retrier.New(time.Second*380, retrier.WithBackoffFactor(1.5), retrier.WithMaxRetries(15, time.Second*30))
+
+			token, err := github.GetGithubToken()
+			if err != nil {
+				return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+			}
+			githubClient, err := github.NewClient(cmd.Context(), token)
+			if err != nil {
+				return fmt.Errorf("setting up Github client: %v", err)
+			}
+
+			if backportVersions[0] == "" {
+				// set up Repo Content Creator handler
+				repoContentManagerOpts := &repoContentManager.Opts{
+					SourceOwner: "rcrozean",
+					SourceRepo:  "eks-distro-build-tooling",
```

Comment:
do you see SourceOwner and SourceRepo being input flags?

either way, let's add constants for the eks-d repo names to the pkg for re-use throughout